jenkins-slave

```diff
@@ -22,10 +22,12 @@
 #  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 #  THE SOFTWARE.
 
-# Usage jenkins-slave.sh [options] -url http://jenkins SECRET SLAVE_NAME
+# Usage jenkins-slave.sh [options] -url http://jenkins [SECRET SLAVE_NAME]
 # Optional environment variables :
 # * JENKINS_TUNNEL : HOST:PORT for a tunnel to route TCP traffic to jenkins host, when jenkins can't be directly accessed over network
 # * JENKINS_URL : alternate jenkins URL
+# * JENKINS_SECRET : agent secret, if not set as an argument
+# * JENKINS_NAME : agent name, if not set as an argument
```

Comment:
`JENKINS_SLAVE_NAME` would be clearer. Or better yet, `JENKINS_AGENT_NAME` with `SLAVE_NAME` replaced by `AGENT_NAME` above.